Modules/TPC/include/TPC/ClusterVisualizer.h

```diff
@@ -65,6 +65,12 @@ class ClusterVisualizer final : public quality_control::postprocessing::PostProc
   /// \param services Interface containing optional interfaces, for example DatabaseInterface
   void finalize(quality_control::postprocessing::Trigger, framework::ServiceRegistryRef) override;
 
+  template <class T>
+  void makeRadialProfile(o2::tpc::CalDet<T>& calDet, TCanvas* canv, int nbinsY, float yMin, float yMax);
```

Comment:
Most probably these functions will never be called outside this code. So the can be in the `private` part of the class.
Also, the binning is not needed here, since it is only taken from `mRanges` via the calDet name. Also, the object should not be modified, so it can be `const.
```suggestion
  void makeRadialProfile(const o2::tpc::CalDet<T>& calDet, TCanvas* canv);
```